docs/testing.md

```diff
@@ -29,9 +29,250 @@ Tests can be launched manually or while using CI (Continuous Integration).
 * [CI] to output a xml JUnit test report, use the option : `--junit-xml=pytest_report.xml`
 * To create code coverage data : `coverage run -m pytest ./tests` then `coverage report` to see the code coverage result or `coverage xml` to output a .xml report file
 
-## Writing tests
-The main idea is to create one test file per source module (eg.: *tests/pipelines/test_pipelines.py* contains all the tests for the module `narps_open.pipelines`).
+## Writing unit tests
```

Comment:
Reading this document, I've just noticed a small typo  "Dependancies" > "Dependencies"